ci/Dockerfile-envoy

```diff
@@ -28,7 +28,7 @@ RUN mkdir -p /etc/envoy
 
 ARG ENVOY_BINARY_SUFFIX=_stripped
 ADD ${TARGETPLATFORM}/build_release${ENVOY_BINARY_SUFFIX}/* /usr/local/bin/
-ADD configs/google_com_proxy.v2.yaml /etc/envoy/envoy.yaml
+ADD configs/google_com_proxy.v3.yaml /etc/envoy/envoy.yaml
```

Comment:
Thoughts on just removing the "v3" postfix from these file names as long as we are changing all of this? I don't see us moving off of v3 for a very long time and I'm not sure it's worth the differentiation in the filename. WDYT?

Reply:
it  would make at least one part of updating configs  in the future easier 

im wondering if it would be better to do in a separate PR, as not 100% of the implications, but can go back through and remove if you think we should do it now

Reply:
Personally I think we should just do it now. @htuch any opinions here?

Reply:
eg - there are docs etc pointing to these files

Reply:
@mattklein123 cool - lets do it now - on it...

Reply:
This is fine.